samples/glean/src/main/java/org/mozilla/samples/glean/GleanApplication.kt

```diff
@@ -22,6 +23,9 @@ class GleanApplication : Application() {
         // Initialize the Glean library. Ideally, this is the first thing that
         // must be done right after enabling logging.
         Glean.initialize(applicationContext)
+        // Initialize the Experiments library right afterwards. Experiments can
+        // not be activated before this, so it's important to do this early.
+        Experiments.initialize(applicationContext)
```

Comment:
Let's also update the sample `README.md` file to mention that we're also showcasing the experiments library here.